src/components/Person/Person.jsx

```diff
@@ -1 +1,27 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+      {age && (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      )}
+      {isMarried ? (
+        <p className="Person__partner">
+          {`${partnerName} is my ${sex === 'm' ? 'wife' : 'husband'}`}
```

Comment:
Let's create a variable for wife/husband to make your code a bit cleaner)
![image](https://github.com/mate-academy/react_person/assets/96409563/3de6c5eb-47b6-4435-9158-054ef42cc4ef)